pkg/build/oci/oci.go

```diff
@@ -119,14 +129,50 @@ func buildImageFromLayer(layerTarGZ string, ic types.ImageConfiguration, created
 
 	v1Image, err = mutate.ConfigFile(v1Image, cfg)
 	if err != nil {
-		return empty.Image, fmt.Errorf("unable to update OCI config file: %w", err)
+		return nil, fmt.Errorf("unable to update OCI config file: %w", err)
 	}
 
-	return v1Image, nil
+	si := signed.Image(v1Image)
+
+	// Attach the SBOM, e.g.
+	if len(sbomFormats) > 0 {
+		var mt ggcrtypes.MediaType
+		var path string
+		switch sbomFormats[0] {
```

Comment:
Currently cosign's method for associating SBOMs supports a single type of SBOM, so I simply pick the first when multiple are emitted.  This can be controlled by restricting or reordering the sbom formats.